code/modules/clothing/gloves/special.dm

```diff
@@ -102,7 +103,7 @@
 	greyscale_colors = null
 	siemens_coefficient = 0.3
 	armor_type = /datum/armor/latex_gloves
-	clothing_traits = list(TRAIT_QUICK_CARRY, TRAIT_FINGERPRINT_PASSTHROUGH)
```

Comment:
Nitrile gloves don't have the `TRAIT_FINGERPRINT_PASSTHROUGH` trait in their `clothing_traits` variable by the by: https://github.com/tgstation/tgstation/blob/be9069ebc38732b8dc3061fc42c23ce437dc084c/code/modules/clothing/gloves/special.dm#L117

Reply:
ah I was under the impression they were just a subtype of latex and were mostly the same thing. I'll update it